src/daemon/ca/manager.rs

```diff
@@ -1219,7 +1281,6 @@ impl CaManager {
         &self,
         ca_handle: &Handle,
         repository: &rfc8183::RepositoryResponse,
```

Comment:
What is this change about?

Reply:
Not see clear from the diff. The implementation for sending RFC8181 messages did not change really, but things got moved around a bit because the embedded option is no longer supported.